src/Summoner/Project.hs

```diff
@@ -115,8 +115,11 @@ generateProject projectName Config{..} = do
         -- Make a commit and push it.
```

Comment:
This comment is also outdated.

Oh, as well as the one above. Could you please update the comments in this function to show what exactly each part do while you're on it?